src/components/TablePopover/TableSortIcon.tsx

```diff
@@ -0,0 +1,63 @@
+import styled from "@emotion/styled";
+import Icon from "@leafygreen-ui/icon";
+import IconButton from "@leafygreen-ui/icon-button";
+import { palette } from "@leafygreen-ui/palette";
+import { size } from "constants/tokens";
+
+const { blue, gray } = palette;
+
+enum SortState {
```

Comment:
[nit] Perhaps we should call this something like `TableSortState` so it's clear it refers to react-table values rather than what Evergreen expects